### ISSUE TITLE
Fields order for package completion

### ISSUE THREAD

**Issue by ncannasse** [vshaxe/vshaxe#375]
When displaying package content, there's a very weird behavior related to completion server.

When starting the first time, completion on a package will scroll down to the classes:

![image](https://user-images.githubusercontent.com/1022912/63957517-d0442280-ca88-11e9-9396-c87e94242f15.png)

After typing one `h3d.anim.` however the other completions will correctly display the following:

![image](https://user-images.githubusercontent.com/1022912/63957557-e520b600-ca88-11e9-8340-f9e9f81abeb0.png)

**Comment by Gama11** [comment]
Does `"editor.suggestSelection": "first"` help with that?

**Comment by Simn** [comment]
I have `"editor.suggestSelection": "first"` in my settings because by default vscode does some retarded sort... memorize... whatever thing.

**Comment by nadako** [comment]
We should put this in the README, because this settings gives better completion :)

**Comment by Gama11** [comment]
Readme would be a bit out of place, but it could definitely be mentioned in https://github.com/vshaxe/vshaxe/wiki/Completion.

**Comment by Simn** [comment]
I think a general "Best practices" or "Things you may want to consider" section in the readme would be good.

**Comment by ncannasse** [comment]
Yes, `"editor.suggestSelection": "first"` fixes it, thanks and yes I think this should be in README so leaving it open.
